src/content/docs/en/guides/images.mdx

```diff
@@ -89,197 +86,9 @@ import myImage from '../assets/my_image.png'; // Image is 1600x900
 
-Widths that are larger than the original image will be ignored to avoid upscaling the image.
-
-```astro
----
-import { Image } from 'astro:assets';
-import myImage from '../assets/my_image.png'; // Image is 1600x900
----
-<Image
-  src={myImage}
-  widths={[240, 540, 720, myImage.width]}
-  sizes={`(max-width: 360px) 240px, (max-width: 720px) 540px, (max-width: 1600px) 720px, ${myImage.width}px`}
-  alt="A description of my image."
-/>
-```
-
-```html
-<!-- Output -->
-<img
-  src="/_astro/my_image.hash.webp"
-  srcset="
-    /_astro/my_image.hash.webp 240w,
-    /_astro/my_image.hash.webp 540w,
-    /_astro/my_image.hash.webp 720w,
-		/_astro/my_image.hash.webp 1600w
-  "
-  sizes="
-    (max-width: 360px) 240px,
-    (max-width: 720px) 540px,
-    (max-width: 1600px) 720px,
-    1600px
-  "
-  alt="A description of my image."
-  width="1600"
-  height="900"
-  loading="lazy"
-  decoding="async"
-/>
-```
-
-##### format
-
-You can optionally state the [image file type](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types#common_image_file_types) output to be used.
-
-By default, the `<Image />` component will produce a `.webp` file.
-
-##### quality
-
-`quality` is an optional property that can either be:
-- a preset (`low`, `mid`, `high`, `max`) that is automatically normalized between formats.
-- a number from `0` to `100` (interpreted differently between formats).
-
-##### inferSize
-
-<p><Since v="4.4.0" /></p>
-
-Allows you to set the original `width` and `height` of a remote image automatically.
-
-By default, this value is set to `false` and you must manually specify both dimensions for your remote image.
-
-Add `inferSize` to the `<Image />` component (or `inferSize: true` to `getImage()`) to infer these values from the image content when fetched. This is helpful if you don't know the dimensions of the remote image, or if they might change:
-
-```astro mark="inferSize"
----
-import { Image } from 'astro:assets';
----
-<Image src="https://example.com/cat.png" inferSize alt="A cat sleeping in the sun." />
-```
-
-`inferSize` can fetch the dimensions of a [remote image from a domain that has not been authorized](#authorizing-remote-images), however the image itself will remain unprocessed.
-
-###### inferRemoteSize()
-
-<p><Since v="4.12.0" /></p>
-
-A function to infer the dimensions of remote images. This can be used as an alternative to passing the `inferSize` property.
-
-```ts
-import { inferRemoteSize } from 'astro:assets';
-const {width, height} = await inferRemoteSize("https://example.com/cat.png");
-```
-
-##### Additional properties
-
-In addition to the properties above, the `<Image />` component accepts all properties accepted by the HTML `<img>` tag.
+Additionally, the `<Image />` component accepts all properties accepted by the HTML `<img>` tag.
 
 For example, you can provide a `class` to the final `<img>` element.
```

Comment:
I wonder if `Additionally` still makes sense now that we're not listing properties anymore.
```suggestion
The `<Image />` component accepts all properties accepted by the HTML `<img>` tag. For example, you can provide a `class` to the final `<img>` element.
```
I guess it follows on from the `ReadMore`... but scanning the page at first glance I didn't see a logical continuation with the previous paragraph. So, I figure `In addition`/`Additionally` is no longer useful.

Reply:
Yes I think it's a question of perception. So your explanation makes sense.
For me, the `ReadMore` with the icon makes the sentence more of an "aside" than an extension so I attach the paragraph to the one before.
So I'm fine with keeping it as is, I just wanted to note it "just in case".

Reply:
Yup, I get that! Let me take one more shot at it for removing weirdness :smile:

Reply:
@ArmandPhilippot one last try?  :smile:

Reply:
@sarah11918  The flow seems better to me like this, thank you! 🙌🏽